### DISCUSSION TITLE
Model ObjectBeamGeometricallyExact2D beams as rigid in the axial direction

### DISCUSSION THREAD

**Discussion by SimonAndreas** [jgerstmayr/EXUDYN#85] [Q&A on usage of Exudyn]
Hi,

I am modelling two beams using the ObjectBeamGeometricallyExact2D element. I would like to model the axial stiffness as rigid. Is this possible? If so, how? 

I want to model them as rigid in the axial direction, because the two beams are connected through a spring in the axial direction. The dynamics in the axial direction are predominantly determined by the spring stiffness, because the axial stiffness of the beams is really high.

I suspect that the high axial stiffness of the beams is resulting in high frequency oscillations, which causes stability issues for the solver. These issues can be resolved if the axial stiffness would be rigid

Thank you in advance

**Comment by SimonAndreas** [comment]
I believe that I managed to get rid of the instabilities by adding some axial damping to my 2Dbeam elements

**Comment by jgerstmayr** [comment]
That sounds good. You can also increase numerical damping of generalized-alpha solver (using a lower spectral radius).

However, a rigid behavior in axial direction is not possible with the current implementation.
